config/default.yml

```diff
@@ -3624,6 +3624,13 @@ Style/PerlBackrefs:
   Enabled: true
   VersionAdded: '0.13'
 
+Style/PreferInterpolation:
```

Comment:
In general it's best to have more neural cop names, as most cops tend to be configurable in the end.